src/invidious/jobs/instance_refresh_job.cr

```diff
@@ -0,0 +1,97 @@
+class Invidious::Jobs::InstanceListRefreshJob < Invidious::Jobs::BaseJob
+  # We update the internals of a constant as so it can be accessed from anywhere
+  # within the codebase
+  #
+  # "INSTANCES" => Array(Tuple(String, String))  # region, instance
```

Comment:
This wasn't working, I guess?

Reply:
What do you mean?

Reply:
Oh, I though it was some commented code, sorry ^^>